src/discord/interaction_handlers/application_commands.rs

```diff
@@ -630,6 +671,7 @@ fn format_player(content: Option<String>, player: &Player) -> InteractionRespons
         data: Some(InteractionResponseData {
             content,
             embeds: Some(embeds),
+            flags: Some(MessageFlags::EPHEMERAL),
```

Comment:
kind of a driveby that i learned about while figuring out how to do this - this kind of read-only status info is nicer if it only shows for the requester and doesn't clutter things up